src/applications/edu-benefits/1995/config/chapters.js

```diff
@@ -17,6 +17,7 @@ import { showSchoolAddress } from '../../utils/helpers';
 import { displayActiveDutyStem } from '../helpers';
 
 import { activeDuty, benefitSelection, stem } from '../pages';
+import { validateWhiteSpace } from '../../../../platform/forms/validations';
```

Comment:
You should be able to just use the relative link /platform/forms/validations

Reply:
Relative link does not work.

Reply:
My bad, platform/forms/validations